# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/nsDocumentViewer.cpp
# Commit: bda050135c64
# Full Hash: bda050135c64c3e4dc8c69b2be06f495592efc6e
# Author: Botond Ballo <botond@mozilla.com>
# Date: 2020-05-06 09:37:16
# Regressor Bug: 1556556
# File Overlap Count: 1
# Description:
#   Bug 1556556 - Move GetCallbackTransform() into a new ViewportUtils class. r=kats
#   
#   This function (and helper functions that wrap it) will be used extensively
#   throughout layout code, so keeping it in APZCCallbackHelper seems awkward.
#   
# ==============================================================================

diff -r 8206cf323d5b -r bda050135c64 layout/base/nsDocumentViewer.cpp
--- a/layout/base/nsDocumentViewer.cpp	Tue May 05 19:22:00 2020 +0000
+++ b/layout/base/nsDocumentViewer.cpp	Tue May 05 19:22:12 2020 +0000
@@ -19,6 +19,7 @@
 #include "nsIContent.h"
 #include "nsIContentViewer.h"
 #include "nsIDocumentViewerPrint.h"
+#include "nsIScreen.h"
 #include "mozilla/dom/BrowsingContext.h"
 #include "mozilla/dom/BeforeUnloadEvent.h"
 #include "mozilla/dom/PopupBlocker.h"
